lib/Db/MessageMapper.php

```diff
@@ -1110,10 +1110,11 @@ public function deleteOrphans(): void {
 		$result->closeCursor();
 
 		$qb2 = $this->db->getQueryBuilder();
+		$query = $qb2
+			->delete($this->getTableName())
+			->where($qb2->expr()->in('id', $qb2->createParameter('ids'), IQueryBuilder::PARAM_INT_ARRAY));
 		foreach (array_chunk($ids, 1000) as $chunk) {
-			$query = $qb2
-				->delete($this->getTableName())
-				->where($qb2->expr()->in('id', $qb2->createNamedParameter($chunk, IQueryBuilder::PARAM_INT_ARRAY), IQueryBuilder::PARAM_INT_ARRAY));
+			$query->setParameter($chunk, IQueryBuilder::PARAM_INT_ARRAY);
```

Comment:
```suggestion
			$query->setParameter('ids', $chunk, IQueryBuilder::PARAM_INT_ARRAY);
```

and give this a quick test :)